test.disabled/ejabberd_tests/tests/muc_SUITE.erl

```diff
@@ -4286,6 +4412,80 @@ parse_result_query(#xmlel{name = <<"query">>, children = Children}) ->
              last = Last,
              count = Count}.
 
+%%--------------------------------------------------------------------
+%%  Room registration use cases
+%%  Tests for race condition that occurs when multiple users
+%%  attempt to start and/or register the same room at the same time
+%%--------------------------------------------------------------------
+load_already_registered_permanent_rooms(Config) ->
+    Room = <<"testroom1">>,
+    Host = <<"localhost">>,
+    ServerHost = <<"localhost">>,
+    Access = none,
+    HistorySize = 10,
+    RoomShaper = none,
+    HttpAuthPool = none,
+
+    %% Write a permanent room
+    F1 = fun() ->
+             mnesia:write(#muc_room{name_host = {Room, Host}, opts = []})
```

Comment:
Is there a way of doing that without redeclaring the record? This way we end up with two definitions that have to be maintained in sync.